mina.netty/src/main/java/org/kaazing/mina/core/service/AbstractIoAcceptorEx.java

```diff
@@ -59,20 +59,21 @@ public BindFuture bindAsync(final SocketAddress localAddress) {
         if (getHandler() == null) {
             throw new IllegalStateException("handler is not set.");
         }
+
         BindFuture bound = bindAsyncInternal(localAddress);
```

Comment:
Surprisingly, the above was not wrapped in synchronized(bindLock) from the beginning.